basiclibrary/src/main/java/basiclibrary/Library.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package basiclibrary;
+
+import java.util.Arrays;
+
```

Comment:
remove new line

Reply:
So when I try and remove this new line, It just reinserts it with the save actions. I have it set up to optimize imports - but its still doing that even when I turn the import optimization off. Ill try to fix it for this resubmit, but  no promises that I will remember in the future.